pkg/utils/release/controller.go

```diff
@@ -44,6 +45,33 @@ func (s *SuiteController) CreateSnapshot(name string, namespace string, applicat
 	return snapshot, s.KubeRest().Create(context.TODO(), snapshot)
 }
 
+func (s *SuiteController) GetSnapshotInNamespace(namespace, componentName string) (*appstudioApi.Snapshot, error) {
+	snapshot := &appstudioApi.SnapshotList{}
+	opts := []client.ListOption{
+		client.MatchingLabels{
+			"test.appstudio.openshift.io/component": componentName,
+		},
+		client.InNamespace(namespace),
+	}
+	err := s.KubeRest().List(context.TODO(), snapshot, opts...)
+
+	if err == nil && len(snapshot.Items) > 0 {
+		return &snapshot.Items[0], nil
+	}
+	return nil, err
+}
+
+func (s *SuiteController) AddLabelToSnapshot(snapshot *appstudioApi.Snapshot) (*appstudioApi.Snapshot, error) {
```

Comment:
could use a comment describing the function

Reply:
Done